src/cascadia/TerminalCore/Terminal.hpp

```diff
@@ -356,27 +352,27 @@ class Microsoft::Terminal::Core::Terminal final :
         til::point pivot;
     };
     std::optional<SelectionAnchors> _selection;

```

Comment:
```suggestion
    std::optional<SelectionAnchors> _selection = std::nullopt;
```
this one also didn't get carried over from the cpp.

Reply:
optionals are technically .. ah you probably talked about this